.github/actions/latest-version/action.yml

```diff
@@ -0,0 +1,85 @@
+# Usage example:
+#
+# - steps:
+#   - id: latest_version
+#     uses: ./.github/actions/latest-version
+#     with:
+#       tarantool-series: '1.10' # don't miss quotes!
+#
+#   - run: echo ${{ steps.latest-version.outputs.package }}
+#   # ---> 1.10.13.0.g1d2c5aad5-1
+#   - run: echo ${{ steps.latest-version.outputs.abc }}
+#   # ---> 1.10.13
+#   - run: echo ${{ steps.latest-version.outputs.abcd }}
+#   # ---> 1.10.13.0
+#   - run: echo ${{ steps.latest-version.outputs.abc-d }}
+#   # ---> 1.10.13-0
+#   - run: echo ${{ steps.latest-version.outputs.git-describe }}
+#   # ---> 1.10.13-0-g1d2c5aad5
+
+name: 'Latest tarantool version'
```

Comment:
Is this a test util or it is a part of an action?

Reply:
It is the test util: I need it to don't repeat the testing code much. Should I clarify it somehow?

Reply:
I'm not sure how action repos are organized. Maybe it worth to add a comment to the top.